spec/marketing_only_app/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
+  "app": {
+    "name": {
+      "title": "Name for App: Marketing Only App",
+      "value": "Marketing Only App"
+    },
+    "description": {
+      "title": "Description for App: Marketing Only App",
+      "value": "Bring your Zendesk into your other service"
+    },
+    "long_description": {
+      "title": "Long Description for App: Marketing Only App",
+      "value": "Zendesk for Marketing Only App allows you to integrate your Zendesk Support helpdesk with your other service."
+    },
+    "installation_instructions": {
+      "title": "Installation Instructions for App: Marketing Only App",
+      "value": "Visit the Plugin Directory [here](http://wordpress.org/extend/plugins/zendesk/installation/) to download and install the plugin."
```

Comment:
Our [guideline](https://support.zendesk.com/hc/en-us/articles/203995373-Basic-writing-guidelines) for links is to use the title or a description as the anchor text, rather than an empty phrase such as *click here*. Also when linking to an external website, include the name of the website. Some users don’t like being redirected to a different website without warning.

Suggested edit: Place the link on "Plugin Directory" and reword as follows:

> Visit the [Plugin Directory](http://wordpress.org/extend/plugins/zendesk/installation/) on the WordPress website to download and install the plugin.